src/main/scala/viper/gobra/frontend/PackageResolver.scala

```diff
@@ -180,16 +183,39 @@ object PackageResolver {
   }
 
   /**
-    * Returns all source files with file extension 'gobraExtension' or 'goExtension in the input resource
+    * Determines if a file is to be imported when the package implemented in the same directory is imported.
+    * All test files (ending in "_test.go") are skipped, as well as all files whose path contains the `testdata` directory.
+    * One difference between our implementation in Gobra and what the Go compiler does is that Gobra does not complain
+    * if the package clause in a test file mentions a package other than the one implemented in the same directory or that package
+    * suffixed with "_test".
+    * TODO: add this check
+    * More information about this can be found in https://tip.golang.org/cmd/go/#hdr-Test_packages.
+    */
+  private def shouldIgnoreResource(r: InputResource): Boolean = {
+    // Here, we could chose either a full path
```

Comment:
```suggestion
    // Here, we could choose the full path instead
```

Reply:
oops, this was actually meant to be deleted, nice catch!